engine/combat/utility/sos_consideration.py

```diff
@@ -61,11 +67,29 @@ def _character_appraisals(self: Self) -> list[Appraisal]:
                 return []
 
     def calculate_actions(self: Self) -> list[Action]:
-        # if the actor isn't enabled, return no actions
         actions = []
+        # TODO(eein): to give value to boosted appraisals we will just multiply
+        # the value by the boost for now, we can modify this when we work further on
+        # utility.
```

Comment:
Yeah, it depends on vulnerabilities I suppose but iirc it adds a portion of MAtk (+element), so damage-wise it's probably not double the utility.

Unless we are fighting Fleshmancer minions, and the utility goes through the roof :P

Reply:
yep, still kinda sorting out what we want to do, but this will allow us to prioritize boosting for now. 
I'd like to control the boosts with the ReasonerExecutionContext down the road somehow. IE DoW where we need to save at least 1 boost.